gcp/modules/gke-network/main.tf

```diff
@@ -44,17 +43,13 @@ module "gke_network" {
   }
 
   static_ip_region         = "${var.infra_region}"
-  create_static_ip_address = "${var.create_static_ip_address}"
+  create_static_ip_address = false
```

Comment:
So why exactly can't we use static IP with Istio?

Reply:
Because Istio creates the load balancer (via service `istio-ingressgateway` of type `LoadBalancer`) and there's no way we can tell it to use that specific IP address, or to create static one, as we can't modify resources managed by GKE Istio add-on (normally you would do this via service annotation).